manifests/init.pp

```diff
@@ -155,6 +158,7 @@
   $user_hash          = {},
   $configure_firewall = undef,
   $install_java       = $jenkins::params::install_java,
+  $java_package_name  = undef,
```

Comment:
Yeah, I was going to ask about this. The Jenkins install failed on 1.6, so we should probably default to 1.7. But I wasn't sure if there were any OS's that had to use 1.6.

If there's no issue, this could be made a param and inherited.